frontend/src/types.ts

```diff
@@ -1154,13 +1154,16 @@ export interface LegacyRecordingFilters {
     operand?: FilterLogicalOperator
 }
 
+export type RecordingDirection = 'newer' | 'older'
```

Comment:
```suggestion
export type SortDirection = 'newer' | 'older'
```

although would need to edit this everywhere it's referenced obvs

Reply:
(or RecordingSortDirection but that feels overly specific IMHO)